infra/terraform/modules/data_access/organization_events/tests/test_organization_events.py

```diff
@@ -0,0 +1,35 @@
+import boto3
+from botocore.exceptions import ClientError
+import pytest
+
+import organization_events
+
+
+@pytest.mark.usefixtures(
+    "given_the_env_is_set",
+    "given_lambda_is_available",
+)
+def test_event_received_member_added(lambda_client_mock, member_added_event, create_role_arn, payload_bytes):
```

Comment:
The name of this test (and the others) doesn't follow the BDD style. It should read more like a natural sentence, eg: `test_when_event_received_a_member_is_added`